### ISSUE TITLE
Rename launcher to chromium

### ISSUE THREAD

**Issue by inancgumus** [grafana/xk6-browser#460]
We might want to rename `launcher` to `chromium` to be semantically more correct, simpler, ready for future browser support, type-safe, and have better compatibility with Playwright.

The following is what we do at the moment to launch a Chromium browser (despite that we only support Chromium):

```js
import launcher from 'k6/x/browser';

const browser = launcher.launch('chromium');
browser.newPage();
```

We panic if something other than `"chromium"` is passed to the `launch()`. We don't need to panic if we directly import `chromium` from `k6/x/browser` as users can catch the problem before running their scripts (as Playwright):

```js
import { chromium } from 'k6/x/browser';

const browser = chromium.launch();
browser.newPage();
```

We can also be ready for future browser support as follows:

```js
import { chromium, firefox, webkit } from 'k6/x/browser';

const browser = chromium.launch();

browser.newPage();
firefox.launch();
 webkit.launch();
```

**Suggestions to make the change**

1. Remove the `browserName` parameter from `main.go#JSModule.Launch()`.
2. Refactor `main.go#JSModule.Launch()` to remove browser name check and panic.
3. Export `"chromium"` in `main.go#Exports()` to import it in a script as `import { chromium } from 'k6/x/browser'`.
4. Update the `README`.
5. Update the `examples/`.
6. Update the documentation (k6-docs).

**In summary (before/after)**

Before:

```js
import launcher from 'k6/x/browser';

const browser = launcher.launch('chromium')
```

After:

```js
import { chromium } from 'k6/x/browser';

const browser = chromium.launch();
```
